src/store/actions.ts

```diff
@@ -402,17 +402,17 @@ export const setUserSeat = (player: string, dispatch: Function): void => {
 };
 
 export const setWinner = (
-  player: number,
+  winnerArray: number[],
   winAmount: number,
   state: IState,
   dispatch: Function
```

Comment:
I don't fully understand what you mean, can you please elaborate?

Reply:
So what you mean is that `setWinner`, instead of receiving four different arguments, should receive an object? `setWinner` already return `void`. I am a little confused, can you please provide a little more detail? Thank you.

Reply:
Thanks for the clarification. I was not aware of this type pattern. I replaced all of the `dispatch` types throughout the app and created #86.